pandas/core/tools/numeric.py

```diff
@@ -162,7 +162,7 @@ def to_numeric(arg, errors="raise", downcast=None):
 
         if downcast in ("integer", "signed"):
             typecodes = np.typecodes["Integer"]
-        elif downcast == "unsigned" and np.min(values) >= 0:
+        elif downcast == "unsigned" and (len(values) == 0 or np.min(values) >= 0):
```

Comment:
Not a strong preference either way but I believe the original suggestion from @TomAugspurger was to do something like:

```suggestion
        elif downcast == "unsigned" and np.min(values, initial=0) >= 0):
```

Reply:
I actually implemented it that way at first, but it was breaking some of the regression since `initial` was only added to `np.amin` in 1.15.0

Reply:
Good to know. Thanks for the insight